usage/rest-server/src/test/java/brooklyn/rest/resources/CatalogResourceTest.java

```diff
@@ -224,20 +224,26 @@ public void testGetCatalogEntityIconDetails() throws IOException {
     }
 
     private void addTestCatalogItem(String catalogItemId) {
+        addTestCatalogItem(catalogItemId, TEST_VERSION, "brooklyn.entity.nosql.redis.RedisStore");
+    }
+
+    private void addTestCatalogItem(String catalogItemId, String version, String service) {
         String yaml =
-                "brooklyn.catalog:\n"+
```

Comment:
Why change the indents? 8 is commonly used for lines that wrap, and is the default recommended by Oracle.